### PR TITLE
User public uri for prx story, not with authorization in it

### PR DESCRIPTION
**Pull request by kookster**
- [x] Make sure there is not an /authorization/ in the prx uri

### CODE REVIEW

File: app/models/episode_story_handler.rb [hunk 1]
```diff
@@ -33,13 +33,21 @@ def update_from_story!(story)
...
 
     update_attributes
     update_audio
     update_image
   end
 
+  def clean_prx_uri(url)
+    result = url
+    if result && !result.match(/authorization/)
+      result = result.gsub('/', '/authorization/')
```
Comment (pkarman) [inline app/models/episode_story_handler.rb]: this line replaces every `/` in the `url` with `/authorization/` - that isn't what I would expect a method called `clean_prx_uri` to do. Likewise, the name of this PR is "not with authorization in it" and this method is *adding* the `authorization`.

So I'm confused.

Comment (kookster) [reply app/models/episode_story_handler.rb]: me too, not sure how my test passed locally. updating.

Comment (kookster) [reply app/models/episode_story_handler.rb]: looks like `guard` was misbehaving - ok, fixing.

**Comment by pkarman** [comment]
looks great @kookster - much clearer.

**Pull request merged**